src/demo/pages/PickersPage/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -93,6 +101,16 @@ export class PeoplePickerTypesExample extends React.Component<any, IPeoplePicker
     return (
       <div>
         { currentPicker }
+        <label> Result Delay in Seconds </label>
+        <select
+          ref={ this._resolveRef('_select') }
+          onChange={() => { this.setState({ promiseDelay: parseInt(this._select.value, 10) }); } }>
```

Comment:
this is hard to read and understand intent.. and why are you using a select and not a Dropdown